package/unios_2.x.sh

```diff
@@ -117,4 +117,186 @@ _tailscale_uninstall() {
 
     systemctl disable tailscale-install.timer || true
     rm -f /lib/systemd/system/tailscale-install.timer || true
+    
+    systemctl disable tailscale-cert-renewal.timer || true
+    systemctl stop tailscale-cert-renewal.timer || true
+    rm -f /lib/systemd/system/tailscale-cert-renewal.service || true
+    rm -f /lib/systemd/system/tailscale-cert-renewal.timer || true
+}
+
+_tailscale_cert() {
+    action="${1:-help}"
+    hostname="${2:-$(hostname)}"
+    cert_dir="${TAILSCALE_ROOT}/certs"
+    
+    case "$action" in
+        generate)
+            if ! _tailscale_is_running; then
+                echo "Tailscale is not running. Please start Tailscale first."
+                exit 1
+            fi
+            
+            mkdir -p "$cert_dir"
+            echo "Generating certificate for $hostname..."
+            
+            if tailscale cert --cert-file "$cert_dir/$hostname.crt" --key-file "$cert_dir/$hostname.key" "$hostname"; then
```

Comment:
I wonder if we can use `tailscale status --peers=false --json | jq -r '.Self.DNSName[:-1]'` to get the hostname automatically, since that is the only valid input anyway.

Reply:
Done! Used `sed 's/\.$//'` instead of `[:-1]` for better shell compatibility, but same result.